pkg/config/mesh/networks_watcher.go

```diff
@@ -63,9 +63,9 @@ func NewNetworksWatcher(fileWatcher filewatcher.FileWatcher, filename string) (N
 		return nil, fmt.Errorf("failed to read mesh networks configuration from %q: %v", filename, err)
 	}
 
-	log.Infof("mesh networks configuration %s", spew.Sdump(meshNetworks))
 	ResolveHostsInNetworksConfig(meshNetworks)
-	log.Infof("mesh networks configuration post-resolution %s", spew.Sdump(meshNetworks))
+	networksdump, _ := (&jsonpb.Marshaler{}).MarshalToString(meshNetworks)
```

Comment:
Just curious, why not use `MeshNetworks.String()`

Reply:
Do you need to specify an indent to make the output multi-line for readability?

Reply:
Yeah I'll do this. should be a bit more clear